src/components/02-components/card-experiences-list/card-experiences-list.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Paper } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CardExperiences from '../card-expreciences/card-experiences';
+
+
+export type BrandProps = {
+  cardList: Array<object>,
+};
+
+export default function CardExperiencesList ({ 
+  cardList,
+}: BrandProps) {
+
+  const theme = useTheme();
+
+  // Styles.
+  const containerStyle = {
+    padding: theme.spacing(5), 
```

Comment:
Use `p` instead of `padding`. It's good to be consistent and avoid mixing different standards.